packages/plugin-eslint/src/lib/__snapshots__/runner.integration.test.ts.snap

```diff
@@ -372,7 +372,7 @@ exports[`executeRunner > should execute ESLint and create audit results for Reac
     "details": {
       "issues": [
         {
-          "message": "Missing \\"key\\" prop for element in iterator",
+          "message": "Missing "key" prop for element in iterator",
```

Comment:
Shouldn't it be escaped? `\"`

Reply:
Vitest no longer escapes quotes in snapshots: https://vitest.dev/guide/migration.html#snapshots-update-3961